src/main.rs

```diff
@@ -131,6 +134,53 @@ pub enum CreateCmd {
     },
 }
 
+#[derive(Debug, StructOpt)]
+/// Create or Redeem from an HTLC address
+pub enum HtlcCmd {
+    /// Creates a new HTLC address with a specified hashlock and timelock (in block height), and transfers a value of tokens to it
+    Create {
+        /// Wallet to use as the payer
+        #[structopt(short = "f", long = "file", default_value = "wallet.key")]
+        files: Vec<PathBuf>,
+
+        /// The base58 address of the intended payee for this HTLC
```

Comment:
I don't think we refer to base58 addresses like this elsewhere? We just say "address"